server/routes/temporary-accommodation/manage.ts

```diff
@@ -302,6 +302,10 @@ export default function routes(controllers: Controllers, services: Services, rou
     ],
   })
 
+  get(paths.bookings.index(), bookingsController.index(), { auditEvent: 'VIEW_BOOKINGS_INDEX' })
+
+  get(paths.bookings.index(), bookingsController.index(), { auditEvent: 'VIEW_BOOKINGS_INDEX' })
```

Comment:
Unnecessary duplication?

Could also be handled by a generic redirects controller as mentioned earlier?